index.html

```diff
@@ -395,30 +395,33 @@ <h3 class="subtitle is-5">Don't hesitate, ask anything</h3>
             <fieldset class="field">
               <label class="label" for="username">Username</label>
               <div class="control has-icons-left has-icons-right">
-                <input id="username" class="input is-success" type="text" name="username">
+                <input id="username" class="input is-success" type="text" name="username"
+                  placeholder="Enter your username" aria-describedby="usernameHelp">
```

Comment:
As far I understand `aria-placeholder` we should use for cases when we can't use regular `placeholder`, so it's redundant here.

Reply:
Yes, this is correct